Playground/templates.json

```diff
@@ -23,6 +23,42 @@
     "insertText" : "var ground = BABYLON.MeshBuilder.CreateGround(\"${1:ground}\", {width: ${2:6}, height: ${3:6}}, scene);",
     "language" : "javascript"
   },
+  {
+    "label" : "Create a point light",
+    "documentation" : "https://doc.babylonjs.com/babylon101/lights#the-point-light",
+    "insertText" : "var pointLight = new BABYLON.PointLight(\"${1:pointLight}\", ${2:new BABYLON.Vector3(0,5,0)}, scene);",

```

Comment:
maybe we could add tab over x,y,z for the position

Reply:
Yeah this is a fantastic idea. Updated in the latest